src/osdatahub/PlacesAPI/places_api.py

```diff
@@ -77,7 +77,7 @@ def query(
             n_required = min(limit, 100)
             while n_required > 0 and data.grown:
                 params["params"].update({"offset": len(data), "maxresults": n_required})
-                response = osdatahub.post(**params)
+                response = requests.post(**params, proxies=osdatahub.get_proxies())
                 data.extend(self.__format_response(response))
```

Comment:
This seems like a reasonable fix, with the caveat that this will remove the data packet checking that Josh created.